### PR TITLE
Added Scrabble into game list

### PR DESCRIPTION
**Pull request by sofiarm21**
Thanks! Scrabble Game
---

@edm00se ready for review

Submitting this PR means you have:

- [x] searched the [readme](https://github.com/edm00se/awesome-board-games/blob/main/readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/main/formatting.md)
- [ ] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/main/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -874,6 +874,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | --------:|
 |   2 - 6 |      6+  |      10m |
 
+### [Scrabble](https://boardgamegeek.com/boardgame/320/scrabble)
+
+> In this classic word game, players use their seven drawn letter-tiles to form words on the gameboard. Each word laid out earns points based on the commonality of the letters used, with certain board spaces giving bonuses. But a word can only be played if it uses at least one already-played tile or adds to an already-played word. This leads to slightly tactical play, as potential words are rejected because they would give an opponent too much access to the better bonus spaces.
+
+![Scrabble image](https://cf.geekdo-images.com/mVmmntn2oQd0PfFrWBvwIQ__imagepage/img/jMu8xW0e_AZpYaBmZN0IRm04jKQ=/fit-in/900x600/filters:no_upscale():strip_icc()/pic404651.jpg)/pic147435.jpg)
```
Comment (edm00se) [inline readme.md:881]: There looks to be two issues with the image, first that there is extra text appended at the end of the line and second that the image is not at all close to the desired size. I've included a suggestion for fixing this line.

```suggestion
![Scrabble image](https://cf.geekdo-images.com/mVmmntn2oQd0PfFrWBvwIQ__itemrep/img/xtiNZzN_7T6xZ3EPxzgYcBz9Ydk=/fit-in/246x300/filters:strip_icc()/pic404651.jpg)
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update the image link and otherwise this pull request looks good. Thanks!

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

File: readme.md (hunk 1 above)
Comment (sofiarm21) [reply readme.md]: @edm00se Thanks for the suggestion!

**Pull request merged**